app/controllers/api/account.php

```diff
@@ -2455,6 +2461,27 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                 ->setMessage($messageDoc)
                 ->setRecipients([$phone])
                 ->setProviderType(MESSAGE_TYPE_SMS);
+
+            if (isset($plan['authPhone'])) {
+                $timeLimit = new TimeLimit('organization:{organizationId}', $plan['authPhone'], 30 * 24 * 60 * 60, $dbForProject); // 30 days
+                $timeLimit
+                    ->setParam('{organizationId}', $project->getAttribute('teamId'));
```

Comment:
Inject the `timelimit` resource and use that here instead 
https://github.com/appwrite/appwrite/blob/1.6.x/app/init.php#L1561